src/chart/components/volumes/volumes.component.ts

```diff
@@ -23,8 +23,10 @@ export type VolumeColorResolver = (priceMovement: PriceMovement, colors: FullCha
 
 export class VolumesComponent extends ChartBaseElement {
 	separateVolumes: SeparateVolumesComponent;
+	dynamicObjectsComponent: DynamicObjectsComponent;
```

Comment:
you can declare it via constructor in arguments using `private` keyword for example, and it will be the same but much less code